api/restaurants.json

```diff
@@ -1,74 +0,0 @@
-[
-  {
-    "id": "cafe-luna",
-    "name": "Cafe Luna",
-    "address": "612 Main St, Cambridge, MA 02139",
-    "phonenumber": "(617) 576-3400",
-    "hours": "9AM-2PM",
-    "notes": [
-      "An amazing Saturday morning brunch!",
-      "Long wait, make a reservation.",
-      "Lovely outdoor seating."
-    ]
-  },
-  {
-    "id": "oleana",
-    "name": "Oleana",
-    "address": "134 Hampshire St, Cambridge, MA 02139",
-    "phonenumber": "(617) 661-0505",
-    "hours": "5PM-9:30PM",
-    "notes": [
-      "Best upscale Mediterranean food in the city.",
-      "Service is always gracious and attentive and helpful without being overbearing.",
-      "Hands down, the best outdoor dining in the city is Oleana's secret garden."
-    ]
-  },
-  {
-    "id": "lonestar-taco-bar",
-    "name": "Lonestar Taco Bar",
-    "address": "635 Cambridge St, Cambridge, MA 02141",
-    "phonenumber": "(857) 285-6179",
-    "hours": "11AM-11PM",
-    "notes": [
-      "Definitely go here for delicious food and bomb margs!",
-      "Al pastor tacos make me gooo crrAzzZZyy.",
-      "The guac was OBVIOUSLY homemade and super fresh."
-    ]
-  },
-  {
-    "id": "s&s-restaurant",
-    "name": "S&S Restaurant",
-    "address": "1334 Cambridge St, Cambridge, MA 02139",
-    "phonenumber": "(617) 354-0777",
-    "hours": "11AM-7PM",
-    "notes": [
-      "Cambridge, MA staple!",
-      "Get an order of pickles, you wont regret it.",
-      "Delicious bakery- I always get the lemon poppy seed cookies."
-    ]
-  },
-  {
-    "id": "little-donkey",
-    "name": "Little Donkey",
-    "address": "505 Massachusetts Ave, Cambridge, MA 02139",
-    "phonenumber": "(617) 945-1008",
-    "hours": "5PM-10PM",
-    "notes": [
-      "The tapas were all very delicious and inspired.",
-      "Don't miss the $1 oysters.",
-      "Sit outside and treat yourself to an incredible cocktail!"
-    ]
-  },
-  {
-    "id": "life-alive",
-    "name": "Life Alive",
-    "address": "765 Massachusetts Ave, Cambridge, MA 02139",
-    "phonenumber": "(617) 354-5433",
-    "hours": "8AM-9PM",
-    "notes": [
-      "An eclectic menu with a hippie vibe.",
-      "My all time favorite food place in Boston!",
-      "Vegetarian and vegan oasis."
-    ]
-  }
-]
```

Comment:
Having both the restaurants.json and individual json files for each restaurant is a little redundant. You could either fetch the entire list of restaurants through your restaurants.json and parse the information to find what you're looking for. or you could be specific in your fetches. But having both is a little bit of overkill and could cause confusion for how to go about getting your information.